src/microbe_stage/systems/SiderophoreSystem.cs

```diff
@@ -0,0 +1,117 @@
+﻿namespace Systems;
+
+using System;
+using Components;
+using DefaultEcs;
+using DefaultEcs.System;
+using DefaultEcs.Threading;
+using World = DefaultEcs.World;
+
+/// <summary>
+///   Handles siderophore projectile collisions
+/// </summary>
+[With(typeof(CollisionManagement))]
+[With(typeof(Physics))]
+[With(typeof(TimedLife))]
+[With(typeof(SiderophoreProjectile))]
+[RunsAfter(typeof(PhysicsCollisionManagementSystem))]
+[RuntimeCost(0.5f, false)]
+public sealed class SiderophoreSystem : AEntitySetSystem<float>
+{
+    private static ChunkConfiguration smallIronChunk = SimulationParameters.Instance.GetBiome("default")
+        .Conditions.Chunks["ironSmallChunk"];
+
+    private static Compound iron = SimulationParameters.Instance.GetCompound("iron");
```

Comment:
Registry data store fields should not be static, these should be `private readonly` and not static.

Reply:
I get CS0120 with that

Reply:
Is the processing method still static for the collisions? If it is then make it non-static.